ui/components/layout/cardElements/cardCollection/CardCollection.module.scss

```diff
@@ -13,4 +15,121 @@
+        top: var(--swiper-navigation-top-offset, 50%);
+        width: var(--swiper-navigation-bg-size);
+        height: var(--swiper-navigation-bg-size);
+        margin-top: calc(0px - (var(--swiper-navigation-size) / 2));
+        z-index: 10;
+        cursor: pointer;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        background-color: var(--header-background-high-contrast-colour);
+        border-radius: 50%;
+        color: var(--header-foreground-contrast-colour);
+        opacity: 0;
+        &.swiper-button-disabled {
+            cursor: auto;
+            pointer-events: none;
+        }
+        &.swiper-button-hidden {
+            cursor: auto;
+            pointer-events: none;
+        }
+        .swiper-navigation-disabled & {
+            display: none !important;
+        }
+        svg {
+            width: 100%;
+            height: 100%;
+            object-fit: contain;
+            transform-origin: center;
+        }
+        .swiper-rtl & svg {
+            transform: rotate(180deg);
+        }
+        &:hover {
+            color: var(--brand-colour);
+            background-color: var(--header-background-colour);
+        }
+        }
+        
+        
+        .swiper-button-prev,
+        .swiper-rtl .swiper-button-next {
+        left: var(--swiper-navigation-sides-offset, 10px);
+        right: auto;
+        box-shadow: 2px 2px 3px 0px rgb(0 0 0 / 35%);
+        }
+        .swiper-button-next,
+        .swiper-rtl .swiper-button-prev {
+        right: var(--swiper-navigation-sides-offset, 10px);
+        box-shadow: -2px 2px 3px 0px rgb(0 0 0 / 35%);
+        left: auto;
+        }
+        .swiper-button-lock {
+        display: none;
+        }
+        // button fade start
+        .swiper-initialized .swiper-button-prev, .swiper-initialized .swiper-button-next {
+        opacity: 1;
+        transition: opacity 0.35s ease-in-out;
+        &.swiper-button-disabled {
+            opacity: 0;
+        }
+        }
+        
+        /* Navigation font start */
+        .swiper-button-prev,
+        .swiper-button-next {
+        &:after {
+            font-family: swiper-icons;
+            font-size: var(--swiper-navigation-icon-size);
+            text-transform: none !important;
+            letter-spacing: 0;
+            font-variant: initial;
+            line-height: 1;
+        }
+        }
+        .swiper-button-prev,
+        .swiper-rtl .swiper-button-next {
+        &:after {
+            content: 'prev';
+            padding-right: .2em;
+        }
+        }
+        .swiper-button-next,
+        .swiper-rtl .swiper-button-prev {
+        &:after {
+            content: 'next';
+            padding-left: .2em;
+        }
+        right: var(--swiper-navigation-sides-offset, 10px);
+        left: auto;
+        }
+        /* Navigation font end */
+          
+    }
+}
+
+  
+.swiperSlide {
+    // composes: test;
+    composes: card; 
```

Comment:
Would have like to use `composes: card from '...`, but I wasn't able to make that work with using` @import` above to bring the external class into this file.